pkg/crc/machine/client.go

```diff
@@ -18,11 +19,12 @@ type Client interface {
 }
 
 type client struct {
-	name  string
-	debug bool
+	name        string
+	debug       bool
+	networkMode network.Mode
 }
 
-func NewClient(name string, debug bool) Client {
+func NewClient(name string, debug bool, networkMode network.Mode) Client {
 	return &client{
 		name:  name,
 		debug: debug,
```

Comment:
This is missing a `networkMode: networkMode`,`

Reply:
thanks, fixed.